modules/n1ql/pages/n1ql-language-reference/explainfunction.adoc

```diff
@@ -0,0 +1,109 @@
+= EXPLAIN FUNCTION
+:description: EXPLAIN FUNCTION provides information about the execution plan for the {sqlpp} subqueries or embedded statements defined for a specified user-defined function. 
+:page-topic-type: reference
+:imagesdir: ../../assets/images
+
+// TEMP
+include::partial$n1ql-language-reference/collapsible-style.adoc[]
+
+[abstract]
+{description}
+
+You can supply the name of an inline or external user-defined function.
+
+* For an inline function, EXPLAIN FUNCTION returns the query plans for all of the subqueries present in the function body.
++
+For more information about inline functions, see xref:n1ql-language-reference/userfun.adoc[].
+
+* For an external function, EXPLAIN FUNCTION returns the query plans for all embedded {sqlpp} queries inside the referenced JavaScript body, or the line number on which a N1QL() call appears.
++
+For more information about user-defined functions with JavaScript, see xref:guides:javascript-udfs.adoc[].
+
+The following constraints apply:
+
+* If a user-defined function itself contains other, nested user-defined function executions, EXPLAIN FUNCTION generates the query plan for the specified function only, and not for its nested {sqlpp} queries.
+
+* If an external function defines an alias for a N1QL() call, EXPLAIN FUNCTION cannot return its line number.
+
```

Comment:
Please add this about how the value of line numbers is calculated - "The line numbers are calculated from the beginning of the JavaScript function definition."